setup.py

```diff
@@ -1,32 +1,32 @@
 from setuptools import find_packages, setup
```

Comment:
I think we can delete this whole file? `poetry build` replaces it

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/659f65d46701275c329e6588c6f4351224771d77